internal/engine/engine_state.go

```diff
@@ -45,9 +47,19 @@ func newManifestState(manifest model.Manifest) *manifestState {
 		lastBuild:          BuildStateClean,
 		manifest:           manifest,
 		pendingFileChanges: make(map[string]bool),
+		pod:                unknownPod,
 	}
 }
 
+type Pod struct {
+	Name      string
+	StartedAt time.Time
+	Status    string
```

Comment:
maybe use the PodPhase type and constants?
https://godoc.org/k8s.io/api/core/v1#PodPhase

Reply:
I feel like the UI should be able to represent statuses that show up in `kubectl get pods`, and afaik there's no const with `CrashLoopBackOff`.